scripts/kotlin_generator.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+import json
+import argparse
+import os
+
+parser = argparse.ArgumentParser(description='Create Kotlin file from Json schema')
+parser.add_argument('-s', '--source', dest='json_schema',
+                   help='Json shema file', required=True)
+
+args = parser.parse_args()
+
+# Capitalize also change the next letter.
+def firstLetterUp(str):
+    return str[0].upper() + str[1:]
+
+class Enum(object):
+    """docstring for Enum"""
```

Comment:
can remove this docstring as non informative

Reply:
done